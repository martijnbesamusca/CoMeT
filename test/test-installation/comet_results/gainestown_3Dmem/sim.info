{'cmdline': ['/home/martijn/Documents/uva/thesis/CoMeT/run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_3Dmem',
             '--',
             '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '114c1c70b953',
 'pin_version': 'unknown',
 'rusage': (22.435136,
            1.38461,
            70688,
            0,
            0,
            0,
            39410,
            5,
            0,
            504,
            3456,
            0,
            0,
            0,
            247361,
            121),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_3Dmem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmp61nF67/run_benchmarks'],
 't_elapsed': 26.654569149017334,
 't_start': 1653910770.974583,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmp61nF67/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test']],
 'user': None,
 'vmem': 453079040L}
