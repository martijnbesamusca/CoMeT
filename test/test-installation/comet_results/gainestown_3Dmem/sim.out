                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |   21246392 |          0 |          0 |          0
  Cycles                             |    9702360 |    9702360 |    9702360 |    9702360
  IPC                                |       2.19 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2695100 |    2695100 |    2695100 |    2695100
  Idle time (ns)                     |        -39 |    2695100 |    2695100 |    2695100
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |           
  num correct                        |    2597544 |          0 |          0 |          0
  num incorrect                      |      26961 |          0 |          0 |          0
  misprediction rate                 |      1.03% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.27 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |    2890060 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |           
    num accesses                     |    7876047 |          0 |          0 |          0
    num misses                       |        913 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |           
    num accesses                     |        959 |          0 |          0 |          0
    num misses                       |        429 |          0 |          0 |          0
    miss rate                        |     44.73% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |    2890060 |          0 |          0 |          0
    num cache misses                 |        925 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf%
    mpki                             |       0.04 |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |           
    num cache accesses               |    7876047 |          0 |          0 |          0
    num cache misses                 |      49534 |          0 |          0 |          0
    miss rate                        |      0.63% |       inf% |       inf% |       inf%
    mpki                             |       2.33 |        inf |        inf |        inf
  Cache L2                           |            |            |            |           
    num cache accesses               |      50943 |          0 |          0 |          0
    num cache misses                 |      49031 |          0 |          0 |          0
    miss rate                        |     96.25% |       inf% |       inf% |       inf%
    mpki                             |       2.31 |        inf |        inf |        inf
DRAM summary                         |            |            |            |           
  num dram accesses                  |      20827 |      20307 |      20366 |      20369
  average dram access latency (ns)   |      43.63 |      38.04 |      37.55 |      38.13
  average dram queueing delay        |       6.20 |       0.61 |       0.12 |       0.70
  average dram bandwidth utilization |      6.00% |      6.00% |      6.00% |      6.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |      16319 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
